### PR TITLE
rebrand resultError page

### PR DESCRIPTION
**Pull request by myang1010**
What (if any) features are you implementing?
- rebrand result error page

Screenshot
Screenshot 2023-12-06 at 9 04 13 AM

### CODE REVIEW

**Comment by myang1010** [comment]
> @myang1010 Can I see a screenshot of the updates pls?

@sydney-devine Sure, I've included the screenshot in the PR description above. Thanks.

**Review by sydney-devine — APPROVED** [review]
Visual looks good on my end! Thank you!

**Comment by sydney-devine** [comment]
@CalebPena & @ivonne-hernandez ^ adding that light lavender since it's the background on a few different pieces is probably a good idea. At least from my perspective? It's the background on the disclaimer page, and then Ivonne is about to have it on the household members too. I defer to you guys though.

**Review by CalebPena — COMMENTED** [review]
Could you use the `--secondary-background-color` instead of `--light-lavender-background`. You can get it by pulling from [Marianna/734/typography-updates](https://github.com/Gary-Community-Ventures/benefits-calculator/tree/Marianna/734/typography-updates) if you have not done so already.

**Comment by myang1010** [comment]
The reason I included `lightLavenderBackground` is because our current `LandingPage.tsx` line 136 on `Marianna/734/typography-updates` branch is calling this variable, `<CardContent sx={{ backgroundColor: theme.lightLavenderBackground }}>`. 

I think is better to make a change to that branch and I can pull in the change. So my other PRs can also get updated accordingly. Or let me know if we decide to make the change from this PR.

File: src/Assets/styleController.ts [hunk 1]
```diff
@@ -73,6 +79,7 @@ function generateMuiOverides(theme: ITheme) {
   const darkTerraCottaColor = theme.secondaryColor;
   const blackColor = '#2A2B2A';
   const midBlue = theme.midBlueColor;
+  const lavenderColor = '#ECDEED';
```
[the remark this answers is not in the document]
Comment (myang1010) [reply src/Assets/styleController.ts:83]: Seems `lavenderColor` is only applied to buttons. I think it's okay to keep it here.

[the remark this answers is not in the document]
Comment (CalebPena) [reply src/Assets/styleController.ts:77]: Isn't there a `theme.secondaryBackgroundColor` that is the lavender color. Could we just use that?

[the remark this answers is not in the document]
Comment (CalebPena) [reply src/Assets/styleController.ts:77]: I see, that is my bad. Could you add it to the style controller with a more general name that describes what it is used for like `hoverColor`. Thank you.

**Pull request merged**